src/test/java/picard/analysis/CollectInsertSizeMetricsTest.java

```diff
@@ -222,6 +224,222 @@ else if (metrics.READ_GROUP.equals("62A79AAXX100907.3")) {
         }
     }
 
+    @Test
+    public void testNoPlots() throws IOException {
+        final File input = new File(TEST_DATA_DIR, "insert_size_metrics_test.sam");
+        final File outfile   = File.createTempFile("test", ".insert_size_metrics");
+        final File pdf   = File.createTempFile("test", ".pdf");
+        outfile.deleteOnExit();
+        pdf.deleteOnExit();
+        final String[] args = new String[] {
+                "INPUT="  + input.getAbsolutePath(),
+                "OUTPUT=" + outfile.getAbsolutePath(),
+                "LEVEL=SAMPLE",
+                "LEVEL=LIBRARY",
+                "LEVEL=READ_GROUP",
+        };
+        Assert.assertEquals(runPicardCommandLine(args), 0);
+
+        Assert.assertEquals(pdf.length(), 0);
+
+        final MetricsFile<InsertSizeMetrics, Comparable<?>> output = new MetricsFile<InsertSizeMetrics, Comparable<?>>();
+        output.read(new FileReader(outfile));
+
+        for (final InsertSizeMetrics metrics : output.getMetrics()) {
```

Comment:
In general if this sort of block of checks is repeated between different between tests its probably best to extract it into a function.  It helps test readers a lot because it helps clarify which thing is being tested specifically.   No need to go back and change it now though.